Gems/ImGui/Code/Source/ImGuiManager.h

```diff
@@ -26,6 +26,20 @@
 
 namespace ImGui
 {
+    enum class ImGuiActivationBroadcast
+    {
+        ActivationBroadcast,
+        ActivationNotBroadcast,
+        DeactivationBroadcast,
+        DeactivationNotBroadcast
```

Comment:
suggestion: Rather than having the two enum states here for activation and deactivation, I think this is clearer as just...

```
enum class ImGuiStateBroadcast
{
    Broadcast,
    NotBroadcast
}
```

And then the two usages...

```
struct ImGuiBroadcastState
{
    ImGuiStateBroadcast m_activationBroadcastStatus = ImGuiStateBroadcast::NotBroadcast;
    ImGuiStateBroadcast m_deactivationBroadcastStatus = ImGuiStateBroadcast::NotBroadcast;
};
```

As we never want to mix the two right?